src/InfluxDB.LineProtocol/LineProtocolWriter.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace InfluxDB.LineProtocol
+{
+    public class LineProtocolWriter
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        private readonly TextWriter textWriter;
+
+        private LinePosition position = LinePosition.NothingWritten;
+
+        public LineProtocolWriter()
```

Comment:
I think giving more options about capacity is good idea.
I would like to avoid exposing that we are using a string builder for now as I want to experiment with writing to an UTF8 buffer.